src/app/components/bitcoin-transaction-item/bitcoin-transaction-inscription-icon.tsx

```diff
@@ -17,8 +19,8 @@ function InscriptionIcon({ inscription, ...rest }: { inscription: SupportedInscr
     case 'image':
       return (
         <Circle
-          bg={color('accent')}
-          color={color('bg')}
+          bg="stacks"
```

Comment:
Maybe this can be a semantic color? or is `stacks` a specific color?

Reply:
I did create `stacks` as a token to use to replace the old @stacks/ui Stacks `brand` and `accent`, which were the same purple.